stats/stats/src/charts/lines/txns_growth.rs

```diff
@@ -0,0 +1,130 @@
+use super::NewTxns;
+use crate::{
+    cache::Cache,
+    charts::{
+        find_chart,
+        insert::{insert_data_many, DateValue},
+        updater::{get_last_row, get_min_block_blockscout},
+    },
+    UpdateError,
+};
+use async_trait::async_trait;
+use entity::{chart_data, sea_orm_active_enums::ChartType};
+use sea_orm::{prelude::*, QueryOrder, QuerySelect};
+use tokio::sync::Mutex;
+
+#[derive(Debug)]
+pub struct TxnsGrowth {
+    cache: Mutex<Cache<Vec<DateValue>>>,
+}
+
+impl TxnsGrowth {
+    pub fn new(cache: Cache<Vec<DateValue>>) -> Self {
+        Self {
+            cache: Mutex::new(cache),
+        }
+    }
+}
+
+#[async_trait]
+impl crate::Chart for TxnsGrowth {
+    fn name(&self) -> &str {
+        "txnsGrowth"
+    }
+
+    fn chart_type(&self) -> ChartType {
+        ChartType::Line
+    }
+
+    async fn update(
+        &self,
+        db: &DatabaseConnection,
+        blockscout: &DatabaseConnection,
+        full: bool,
+    ) -> Result<(), UpdateError> {
+        let chart_id = find_chart(db, self.name())
+            .await
+            .map_err(UpdateError::StatsDB)?
+            .ok_or_else(|| UpdateError::NotFound(self.name().into()))?;
+        let min_blockscout_block = get_min_block_blockscout(blockscout)
+            .await
+            .map_err(UpdateError::BlockscoutDB)?;
+        let last_row = get_last_row(self, chart_id, min_blockscout_block, db, full).await?;
+        let mut data = {
+            let mut cache = self.cache.lock().await;
+            cache
+                .get_or_update(async move { NewTxns::read_values(blockscout, last_row).await })
```

Comment:
we pass here `last_row` for growth chart, but we should provide it for new txns chart
this can lead to improper updating for new txns, for example if new txns will be updated only once two days, and growth will be updated once every day, then new txns could miss one day

"but something something we probably will never have such config" - yes, but I think we should at least make everything correct but slow, if the config is wrong

I think the easiest solution is to provide here `None` instead of last row, and schedule new txns update before txns growth

Reply:
Agree, sending None and schedule before is ok for me